Userland/Libraries/LibJS/Runtime/Intl/SegmentsPrototype.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022, Idan Horowitz <idan.horowitz@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/GlobalObject.h>
+#include <LibJS/Runtime/Intl/Segments.h>
+#include <LibJS/Runtime/Intl/SegmentsPrototype.h>
+
+namespace JS::Intl {
+
+// 18.5.2 The %SegmentsPrototype% Object, https://tc39.es/ecma402/#sec-%segmentsprototype%-object
+SegmentsPrototype::SegmentsPrototype(GlobalObject& global_object)
+    : PrototypeObject(*global_object.object_prototype())
+{
+}
+
+void SegmentsPrototype::initialize(GlobalObject& global_object)
+{
+    Object::initialize(global_object);
+}
```

Comment:
Do we even need to override this if all we do is calling the base `initialize()`? Not sure how we do this elsewhere.

Reply:
This becomes more filled in a couple of commits later - `LibJS: Implement Intl %SegmentsPrototype%[@@iterator] `